src/main/java/racingcar/model/Winners.java

```diff
@@ -0,0 +1,30 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Winners {
+	private List<String> winners;
```

Comment:
```suggestion
	private List<Car> winners;
```
String보다 Car 객체 자체를 가지도록 구현하는 것은 어떨까?